index.html

```diff
@@ -25,11 +25,188 @@ <h1>Kiprono Japhet</h1>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (4).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+    <div class="projects-wrapper">
+        <div class="project-image">
+            <img class="project-img" src="images/Rectangle 21 (5).png" alt="image">
+        </div>
+        <div class="project-lg">
+          <div class="project-lg-1">
+            <h3>Project name goes here</h3> 
+            
+            <button type="submit" class="project-btn-1"><span class="btn-p">HTML/CSS</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">Ruby on Rails</span></button>
+            <button type="submit" class="project-btn-1"><span class="btn-p">JavaScript</span></button>
+            
+            <button type="submit" id="project-view">
+                <span class="project-view-union">See this project</span>
+                
+                <span><img id="union" src="images/Union.png" alt="union"></span>
+            </button>
+          </div>
+        </div>
+    </div>
+  </section>
+  <section class="about">
+    <div class="about-me">
+        <h1>About</h1>
+        <P>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</P>
+
+        <button type="submit" id="about-view">
+            <span class="project-view-btn">Get my CV</span>
+            
+            <span><img id="cv-btn" src="images/ic_download.png" alt="union"></span>
+        </button>
```

Comment:
- [ ] Let's make sure to implement and follow the button styling: Active, Hover and Pressed
![image](https://user-images.githubusercontent.com/46839548/153730468-bc0fa962-ea92-4855-b369-b1ba4b980587.png)